appengine_config.py

```diff
@@ -70,13 +70,14 @@ def save(self):
 
 # Root path of the app.
 ROOT_PATH = os.path.dirname(__file__)
-
 _PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
-pil_path = os.path.join(_PARENT_DIR, 'oppia_tools', 'PIL-1.1.7')
+oppia_tools_path = os.path.join(_PARENT_DIR, 'oppia_tools')
 
-if not os.path.isdir(pil_path):
-    raise Exception('Invalid path for oppia_tools library: %s' % pil_path)
-sys.path.insert(0, pil_path)
+if os.path.isdir(oppia_tools_path):
```

Comment:
Add a comment above this line:

```
# oppia_tools/ is available locally (in both dev and prod mode). However, on the
# GAE production server, oppia_tools/ is not available, and the default PIL third-party
# library is used instead.
#
# We cannot special-case this using DEV_MODE because it is possible to run Oppia in
# production mode locally, where a built-in PIL won't be available. Hence the check for
# oppia_tools instead.
```